packages/protobuf/src/registry.ts

```diff
@@ -427,7 +427,7 @@ const OPEN: FeatureSet_EnumType.OPEN = 1;
 
 // prettier-ignore
 // bootstrap-inject defaults: EDITION_PROTO2 to EDITION_2023: export const minimumEdition: SupportedEdition = $minimumEdition, maximumEdition: SupportedEdition = $maximumEdition;
-// generated from protoc v29.2
+// generated from protoc v30.0
```

Comment:
We're adding `enforce_naming_style`, but don't advertise support for edition 2024 yet, because it hasn't been released.